features/step_definitions/functional_tests/functional_tests_steps.rb

```diff
@@ -0,0 +1,58 @@
+Given(/^the test package "([^"]*)" is installed on the controller node$/) do |test_package|
+  control_node.rpm_q(test_package)
+end
+
+Given(/^the package "([^"]*)" is installed on the controller node$/) do |client_package|
+  control_node.rpm_q(client_package)
+end
+
+Given(/^the proper cirros test image has been created$/) do
+  test_image_name = "cirros-test-image-uec"
+
+  if control_node.openstack.image.list.find {|img| img.name == test_image_name }
+    control_node.openstack.image.delete(test_image_name)
+  end
+
+  @test_image = control_node.openstack.image.create(
+    test_image_name,
+    copy_from: "http://clouddata.cloud.suse.de/images/cirros-0.3.3-x86_64-uec.tar.gz",
+    container_format: :bare,
+    disk_format: :raw,
```

Comment:
Actually no, will replace the url the one pointing at the `ami` image at `http://clouddata.cloud.suse.de/images/cirros-0.3.3-x86_64-disk.img`